### ISSUE TITLE
start charging

### ISSUE THREAD

**Issue by gertparisis** [tillsteinbach/WeConnect-python#5]
**Describe the bug**
when I alter the control value /vehicles/XXXXXX/controls/charging to "start"
I have to do it twice 
the first attempt throws an error

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/addressable.py", line 236, in value
    self.setValueWithCarTime(newValue=newValue, lastUpdateFromCar=None)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/addressable.py", line 176, in setValueWithCarTime
    self.notify(flags)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/addressable.py", line 53, in notify
    observer(element=self, flags=flags)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/elements.py", line 2121, in __onChargingControlChange
    self.vehicle.updateStatus(force=True)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/elements.py", line 157, in updateStatus
    and cacheDate < (datetime.utcnow() - timedelta(seconds=self.weConnect.maxAge))):
TypeError: unsupported type for timedelta seconds component: NoneType

the 2nd time it works

**To Reproduce**
w=weconnect.WeConnect(username='xxxx',password='xxxx',updateAfterLogin=False, loginOnInit=False)
 w.login()
 w.update()
cc = w.getByAddressString('/vehicles/xxxx/controls/charging')
cc.value = 'start'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/addressable.py", line 236, in value
    self.setValueWithCarTime(newValue=newValue, lastUpdateFromCar=None)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/addressable.py", line 176, in setValueWithCarTime
    self.notify(flags)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/addressable.py", line 53, in notify
    observer(element=self, flags=flags)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/elements.py", line 2121, in __onChargingControlChange
    self.vehicle.updateStatus(force=True)
  File "/Users/gert/Downloads/WeConnect-python-main/weconnect/elements.py", line 157, in updateStatus
    and cacheDate < (datetime.utcnow() - timedelta(seconds=self.weConnect.maxAge))):
TypeError: unsupported type for timedelta seconds component: NoneType
cc.value = 'start'

 - OS: MacOS
 - car : id4

(sorry if this is not a bug, but some code error)

**Comment by tillsteinbach** [comment]
hmmm, yes this is a bug! I'll try to fix that quickly, can you in the meantime try if this this workaround works for you?
```
w=weconnect.WeConnect(username='xxxx',password='xxxx',updateAfterLogin=False, loginOnInit=False, maxAge=0)
```

**Comment by tillsteinbach** [comment]
Can you also check "0.10.0dev2" prerelease? This should solve this issue.

**Comment by tillsteinbach** [comment]
Ah and by the way: It will work with
`cc.value = 'start'`
but the correct way should be
`cc.value = Controls.Operation.START`

I definitiv have to write a bit of documentation ;)

**Comment by gertparisis** [comment]
I now get this,

>>> cc.value='start'
/vehicles/WVGZZZE2ZMP015414: Unknown attribute chargingRequestStatus with value {'status': 'in_progress'}

documentation would be nice, I looked a bit at the cli code and tried a few things to get going ;-)

**Comment by gertparisis** [comment]
and if I try it with cc.value = Controls.Operation.START

I get 

ValueError: id /vehicles/xxxxxx/controls/charging cannot be set to value Controls.Operation.STOP. You need to provide it in the correct format select one of [start, stop]

**Comment by tillsteinbach** [comment]
ah you are right! Another thing to fix!

**Comment by tillsteinbach** [comment]
> I now get this,
> 
> > > > cc.value='start'
> > > > /vehicles/XXXXXX: Unknown attribute chargingRequestStatus with value {'status': 'in_progress'}
> 
> documentation would be nice, I looked a bit at the cli code and tried a few things to get going ;-)

I will cover this under #2 but the car should still do something. this is just the feedback that the API does not yet understand

**Comment by gertparisis** [comment]
Correct, it starts charging as expected
thx for the quick responses!

**Comment by tillsteinbach** [comment]
> Ah and by the way: It will work with
> `cc.value = 'start'`
> but the correct way should be
> `cc.value = Controls.Operation.START`
> 
> I definitiv have to write a bit of documentation ;)

This is now hopefully fixed in 0.10.0dev3

**Comment by tillsteinbach** [comment]
Fixed in https://github.com/tillsteinbach/WeConnect-python/releases/tag/v0.10.0
